src/quicktions/quicktions.pyx

```diff
@@ -70,16 +70,6 @@ cdef pow10(Py_ssize_t i):
 
 # Half-private GCD implementation.
 
-cdef extern from *:
```

Comment:
Please keep this in here. It's strictly an internal implementation detail of the module.

Reply:
I am trying to remember if this being in the pyx caused compilation errors or not, but will test